### PR TITLE
Reformatting code to follow MyBB standards

### PR DESCRIPTION
**Pull request by euantorano**
Also moving test class autoloading to `autoload-dev` in `composer.json`.

### CODE REVIEW

File: src/Generator.php [hunk 1]
```diff
@@ -119,7 +119,7 @@ public function __construct(array $settings = null)
 	 */
 	public function setSecure($secure = true)
 	{
-		$this->secure = (bool) $secure;
+		$this->secure = (bool)$secure;
```
Comment (wpillar) [inline src/Generator.php:122]: Is this one of our standards?!

Comment (euantorano) [inline src/Generator.php:122]: I'm not sure, I cheated by running PHPStorm's auto styler :wink:, but my config isn't quite right (see the Circle CI output).

Comment (wpillar) [inline src/Generator.php:122]: It's not a PSR standard, so technically it's down to developer preference. But my argument would be that a space between the cast and the variable is more consistent with other PSR standards.

Comment (euantorano) [inline src/Generator.php:122]: I would agree. It looks neater with a space too, so I will revert it.

The other issue is with the alignment of PHPDoc comments and variables. Personally I think aligning them is a waste of space and doesn't help much with readability. @JN-Jones likes them though :stuck_out_tongue:

Comment (wpillar) [inline src/Generator.php:122]: Agreed on PHPDoc too.

**Pull request closed without merging**